### PR TITLE
Add net_peerCount RPC method

### PR DESCRIPTION
**Pull request by aunyks**

### CODE REVIEW

**Comment by holgerd77** [comment]
Hi Gerald, very cool that you are doing continued work on this! 👍

**Comment by aunyks** [comment]
Thanks Holger!

File: test/rpc/net/peerCount.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+function createNode (opened = true, commonChain = new Common('mainnet')) {
+  let chain = new Chain({ blockchain: blockChain({}) })
+  chain.opened = true
+  return {
+    services: [
+      {
+        name: 'eth',
+        chain: chain,
+        synchronizer: {
```
Comment (vpulim) [inline test/rpc/net/peerCount.js]: Same here. Suggested change:

```
services: [
      {
        name: 'eth',
        chain: chain,
        pool: { peers: [1, 2, 3] }
      }
```

**Comment by aunyks** [comment]
I think it's the default Prettier config in my VS Code code setup doing that. I'll refactor.

**Comment by aunyks** [comment]
Changes to the non-`net/peerCount` RPC methods are a resolution to errors that arise when creating a test node: they must have a `synchronizer` or `pool` key, otherwise the test will fail due to an erroneously trying to retrieve one of those values.

**Comment by aunyks** [comment]
Should be ready now.

**Pull request merged**